src/EmailAddress.php

```diff
@@ -547,6 +547,9 @@ protected function idnToAscii($email)
      */
     protected function idnToUtf8($email)
     {
+        if(strlen($email) == 0)
+            return $email;
```

Comment:
PSR-2, missing parentheses and space after `if`

http://www.php-fig.org/psr/psr-2/